File: website/app/models/HelpReply.scala
package models

import java.util.Date
import com.fasterxml.jackson.databind.annotation.JsonDeserialize

case class HelpReply(
                      @JsonDeserialize(contentAs = classOf[java.lang.Long])
                      id: Option[Long] = None,

                      requestId: Long,

                      @JsonDeserialize(contentAs = classOf[java.lang.Long])
                      replierId: Option[Long] = None,

                      text: String,
                      creationDatetime: Option[Date] = None)

File: website/public/javascripts/models/help-reply.js
CBR.Models.HelpReply = new Class({
    Extends: CBR.Models.JsonSerializableModel,

    options: {  // Defaults
    },

    getId: function () {
        return this.options.id;
    },

    getRequestId: function () {
        return this.options.requestId;
    },

    getReplierId: function () {
        return this.options.replierId;
    },

    getText: function () {
        return this.options.text;
    },

    getCreationDatetime: function () {
        return this.options.creationDatetime;
    }
});
